astropy/cosmology/core.py

```diff
@@ -37,8 +37,6 @@
 
 __all__ = ["Cosmology", "CosmologyError", "FlatCosmologyMixin"]
 
-__doctest_requires__ = {}  # needed until __getattr__ removed
```

Comment:
LoL I don't even know what "needed until __getattr__ removed" means but CI is passed, so I guess this is good to merge. Thanks!